src/tests/the_if_function.test.js

```diff
@@ -1,5 +1,5 @@
 const _if = require("../challenges/the_if_function.js")
 
 test("true should call func1", () => {
-  expect(_if(true, func1, func2)).toBe(func1())
+  expect(_if(true, func1(), func2())).toBe("function 1 was called")
```

Comment:
The `_if` takes functions, not the return value of a function, so these should not be invoked here.